src/ImageRunModal.jsx

```diff
@@ -635,6 +638,75 @@
         return owner === systemOwner;
     };
 
+    prefillModal() {
+        const container = this.props.container;
+        const containerDetail = this.props.containerDetail;
+        const image = this.props.localImages.find(img => img.Id === container.ImageID);
+        const owner = container.isSystem ? 'system' : this.props.user;
+
+        if (containerDetail.Config.CreateCommand) {
+            this.setState({
+                dialogWarning: _("This container was not created by cockpit"),
+                dialogWarningDetail: _("Some options may not be copied to the new container."),
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)